packages/website/src/components/Playground.module.css

```diff
@@ -39,5 +39,5 @@
   width: 100%;
   height: calc(100% - var(--ifm-navbar-height));
   top: var(--ifm-navbar-height);
-  z-index: var(--ifm-z-index-fixed);
+  z-index: 100;
```

Comment:
This change works but it moves us from a nice known variable to a new hardcoded number. Hardcoded `z-index` values used across different components can become difficult to work with.

How about making this based on the `ifm-z-index-fixed` variable with a `calc`?

```css
calc(var(--ifm-z-index-fixed) - 1);
```

cc @Josh-Cena as our Docusaurus expert in case there's a better pattern ✨

Reply:
I'm not aware of a better pattern than `calc(var(--ifm-z-index-fixed) - 1);` and honestly it feels the same as `100` to me (`1` is still an arbitrary constant).

@lex111 Any thoughts?

Reply:
Yes, using `calc` with the `--ifm-z-index-fixed` looks like a appropriate solution. But maybe just remove the current `z-index` from the `codeContainer`, why is it needed there?

Reply:
IIRC, it's because the tooptip when you hover over a type needs to be above the navbar.